### DISCUSSION TITLE
Setup a page that only returns HTTP 200 code

### DISCUSSION THREAD

**Discussion by cbroomemktw** [vikejs/vike#1705] [Discussion]
I'm trying to deploy a Vike app to AWS but it seems all the pages respond with HTTP 103s when deployed and those responses trip up the health checks. I'm investigating where those 103s come from--they don't pop up locally--but is there a good way to just create a page that returns 200s?

**Comment by brillout** [comment]
It doesn't seem related to Vike, but let me know if you think otherwise. If you don't use [`pageContext.httpResponse.earlyHints`](https://vike.dev/renderPage#usage) then your resopnse doesn't contain any `103 Early Hint`.

**Reply by cbroomemktw** [reply]
Ah, I see it now. There's an earlyHints in the default Vike template. 

I also thought of a simple way around the health check issue: I just created a health.html file in the public folder, however I think these 103s are tripping up the plumbing in our AWS deployment for the other pages.

**Reply by brillout** [reply]
:+1: Let me know if there is anything Vike can do on its side.

Also, in case your company would be up for it: we're looking for [sponsors](https://github.com/vikejs/vike/issues/1350). Any help is appreciated.
